src/api/ZoweExplorerZosmfApi.ts

```diff
@@ -71,6 +71,19 @@ export class ZosmfUssApi extends ZosmfApiCommon implements ZoweExplorerApi.IUss
         return zowe.Upload.fileToUSSFile(this.getSession(), inputFilePath, ussFilePath, binary, localEncoding, task, etag, returnEtag);
     }
 
+    public async putContent(inputFilePath: string, ussFilePath: string,
```

Comment:
@dkelosky, Should we deprecate the `putContents` function in favor of `putContent` here as well?
And maybe combine implementations i.e. have `putContents` create the options object and then call `putContent`, just a thought! 😋

Reply:
I'll add a `@deprecated` mark here and call `putContent` from `putContents` as you suggested.

Reply:
I think this is addressed now